stl/inc/chrono

```diff
@@ -5642,6 +5644,50 @@ namespace chrono {
         return _Os << _STD format(_Os.getloc(), _STATICALLY_WIDEN(_CharT, "{:%T}"), _Val);
     }
 
+#pragma warning(push)
+#pragma warning(disable : 4365) // 'argument': conversion from 'char' to 'const wchar_t', signed/unsigned mismatch
+    template <class _CharT>
+    _NODISCARD decltype(auto) _Widen_string(const string& _Str) { // FIXME, is this ok?
+        if constexpr (is_same_v<_CharT, char>) {
+            return _Str;
+        } else {
+            return wstring{_Str.begin(), _Str.end()};

```

Comment:
As the suppressed warning hints, this isn't calling `ctype::widen` but rather just casts the narrow value to wide. Is that correct here? I only ask because if it is, then the handling of the time zone abbreviation for [time.parse] could be simplified to use this as well. I'm a bit fuzzy on the issue of locale vs. code page and the various failure modes.

Reply:
`ctype::widen` is more likely to be correct. Directly converting `char` to `wchar_t` assumes that the `char` value is valid as a UTF-16 codepoint; this is true for ASCII but not true for other characters in other codepages. If ICU always returns abbreviations that are in ASCII, this quick-and-dirty technique is fine; if ICU can ever return non-ASCII abbreviations then we should be going through `ctype::widen`. If [time.parse] is already using `ctype::widen` I don't think it should change, it's doing the right (but slower, and more code) thing.

Reply:
I'll replace my FIXME with a TRANSITION comment mentioning that ctype::widen should probably be used; even if this ends up being ok, the TRANSITION comment will indicate that it's not a good precedent to follow.